client/src/components/visualisation/ConfigMenu.jsx

```diff
@@ -0,0 +1,253 @@
+import React, { PropTypes } from 'react';
+import DashSelect from '../common/DashSelect';
+import ColumnMenu from './configMenu/ColumnMenu';
+import LabelColumnMenu from './configMenu/LabelColumnMenu';
+import LabelInput from './configMenu/LabelInput';
+
+
+const getDatasetArray = datasetObject => {
+  const datasetArray = [];
+  const sortFunction = (a, b) => a.name.toLowerCase() > b.name.toLowerCase();
+
+  Object.keys(datasetObject).forEach(key => {
+    datasetArray.push(datasetObject[key]);
+  });
+
+  datasetArray.sort(sortFunction);
+
+  return datasetArray;
+};
+
+const getDatasetOptions = datasetArray => {
```

Comment:
Would it be clearer to use `map` instead of `forEach` here? Something like
```js
const getDatasetOptions = datasetArray => (
  datasetArray.map(option => ({
    value: option.id,
    label: option.name,
  }))
);
```